pnpm-workspace.yaml

```diff
@@ -7,6 +7,12 @@ onlyBuiltDependencies:
 
 strictDepBuilds: true
 
+autoInstallPeers: false
+strictPeerDependencies: true
+peerDependencyRules:
+  allowedVersions:
+    astro: ^5
```

Comment:
The actual PR content

Reply:
https://www.npmjs.com/package/@konami-emoji-blast/astro/v/0.2.0 🎆

Reply:
This can be removed now, right?

Reply:
Yup!